addons/web/static/src/legacy/utils.js

```diff
@@ -0,0 +1,277 @@
+/** @odoo-module **/
+
+import { browser } from "../core/browser/browser";
+import AbstractStorageService from "web.AbstractStorageService";
+import { ConnectionAbortedError, RPCError, makeErrorFromResponse } from "../core/network/rpc_service";
+import { ErrorDialog } from "../core/errors/error_dialogs";
+
+export function mapDoActionOptionAPI(legacyOptions) {
+    legacyOptions = Object.assign(legacyOptions || {});
```

Comment:
I don't think the `Object.assign` call does anything useful here.

Reply:
It is useless indeed.